config-service/build.gradle.kts

```diff
@@ -84,16 +84,13 @@ dependencies {
   testFixturesAnnotationProcessor("org.projectlombok:lombok:1.18.12")
   testFixturesCompileOnly("org.projectlombok:lombok:1.18.12")
 
+  runtimeOnly("io.netty:netty-codec-http2:4.1.59.Final")
```

Comment:
Same comment as attribute service - please move these back into constraints block (and indicate the corresponding cause of the constraint)

Reply:
done.